packages/desktop-client/src/hooks/useServerURL.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
+
+import { send } from 'loot-core/src/platform/client/fetch';
+
+export function useServerURL() {
+  let [url, setUrl] = useState('');
+  useEffect(() => {
+    async function run() {
+      let url = (await send('get-server-url')) || '';
```

Comment:
suggestion: `url` is already defined in the upper scope. Ideally this new variable should have a different name.

```suggestion
      let serverUrl = (await send('get-server-url')) || '';
```